content/api/commands/origin.md

```diff
@@ -233,11 +231,41 @@ and the protocol defaults to `https`. When `cy.visit()` is called with the path
 `/history/founder`, the three are concatenated to make
 `https://www.acme.com/history/founder`.
 
+#### Alternative navigation
+
+```js
+// Do things in primary origin...
+
+cy.visit('https://www.acme.com/history/founder')
+
+// The cy.origin block is required to interact with the cross origin page.
+cy.origin('www.acme.com', () => {
+  cy.get('h1').contains('About our Founder, Marvin Acme')
+})
+```
+
+Here the cross origin page is visited prior to the `cy.origin` block, but any
+interactions with the window are performed within the block which can
+communicate with the cross origin page
+
+#### Incorrect usage
+
+```js
+// Do things in primary origin...
+
+cy.visit('https://www.acme.com/history/founder')
+
+// This command will fail, it's executed on localhost but the application is at acme.com
+cy.get('h1').contains('About our Founder, Marvin Acme')
+```
+
+Here this command fails because we are interacting with a cross-origin window
+outside of the cy.origin block, 'localhost' can't communicate with 'acme.com'.
```

Comment:
```suggestion
outside of the `cy.origin()` block because Cypress has not been injected into the 'acme.com' domain.
```

Reply:
the problem really is that localhost can't communicate with acme.com, even if a cypress instance was injected into acme.com localhost still can't communicate outside of post message.

is this better?

```suggestion
outside of the cy.origin block, due to 'same-origin' restrictions, the 'localhost' javascript context can't communicate with 'acme.com'.
```

Reply:
@mjhenkes yes!